### PR TITLE
new specs with correct v3

### PR DESCRIPTION
**Pull request by japrescott**
regenerate the specs with correct v3 config

### CODE REVIEW

**Review by MalteEbner — CHANGES_REQUESTED** [review]
Please remove all `__init__.py-e` files

**Review by MalteEbner — APPROVED** [review]
Generation looks good.

File: tests/api_workflow/test_api_workflow_compute_worker.py
Comment (guarin) [inline tests/api_workflow/test_api_workflow_compute_worker.py:1]: I think there is another test that has to be updated around line 218 (`test_selection_config`). It should also use the new `V3` classes. When merging this PR we should not have to import any of the old `SelectionCon`

File: tests/api_workflow/test_api_workflow_compute_worker.py [hunk 1]
```diff
@@ -43,6 +43,18 @@
     SelectionStrategyType,
     TagData,
 )
+from lightly.openapi_generated.swagger_client.models.selection_config_v3 import (
+    SelectionConfigV3,
+)
```
Comment (guarin) [inline tests/api_workflow/test_api_workflow_compute_worker.py:48]: Sorry, I forgot to mention this, but we usually import the swagger generated classes directly from `swagger_client.models` like this:
```
from lightly.openapi_generated.swagger_client.models import (
    SelectionConfigV3,
    SelectionConfigV3Entry,
    ...
)

It makes the code a bit easier to read, the drawback is that the VSCode auto-import function does not use this format.

**Review by guarin — CHANGES_REQUESTED** [review]
Nice! I think one more tests needs to be changed but otherwise this looks good to me :)

File: tests/api_workflow/test_api_workflow_compute_worker.py [hunk 2]
```diff
@@ -101,16 +101,16 @@ def test_create_compute_worker_config__selection_config_is_class(self) -> None:
...
-                        input=SelectionConfigEntryInput(
+                    SelectionConfigV3Entry(
+                        input=SelectionConfigV3EntryInput(
                             type=SelectionInputType.EMBEDDINGS,
                             dataset_id=utils.generate_id(),
                             tag_name="some-tag-name",
                         ),
-                        strategy=SelectionConfigEntryStrategy(
+                        strategy=SelectionConfigV3EntryStrategy(
                             type=SelectionStrategyType.SIMILARITY,
```
Comment (guarin) [inline tests/api_workflow/test_api_workflow_compute_worker.py:114]: Could you update `SelectionStrategyType` also to `SelectionStrategyTypeV3`?

**Review by guarin — APPROVED** [review]
One last small thing otherwise everything LGTM! I already approved so you can merge whenever you are ready :)

**Pull request merged**